### PR TITLE
FAB-95-API-Integration

### PR DESCRIPTION
**Pull request by OluwatobiOmole**
Some functionality for data transformation was created in the Flask API Integration to test how it interacts with the dataquality checker script and Azure blob. Please let me know what I should change as I am due to add more functions

### CODE REVIEW

**Review by SeanMcCrossanTUD — COMMENTED** [review]
LG so far think it's a good start. We can hop on a call tomorrow to go through what else you need such as initialising flask etc

File: python-da/data-prep/src/app.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import pandas as pd
 from azure.core.exceptions import AzureError
 from azure.storage.blob import BlobServiceClient
+from .data_prep import DataPrep
```
Comment (D22124491-NaveenMaheswaran) [inline python-da/data-prep/src/app.py:7]: is . data_prep correct syntax

**Review by D22124491-NaveenMaheswaran — APPROVED** [review]
just check line 7 syntax otherwise LG

File: python-da/data-prep/src/app.py (hunk 1 above)
Comment (OluwatobiOmole) [reply python-da/data-prep/src/app.py:7]: Will resolve this after our API call thanks

**Pull request closed without merging**